src/Configuration/Config.Xml/test/ConfigurationProviderXmlTest.cs

```diff
@@ -20,6 +20,8 @@ public override void Combine_after_other_provider()
             // Disabled test due to XML handling of empty section.
         }
 
+        protected override bool CanRepresentNulls => false;
```

Comment:
So the fact that not all providers can represent nulls make me wary of making a change like this...